### DISCUSSION TITLE
大数计算功能中的ECC Calc选择x25519曲线怎么计算？

### DISCUSSION THREAD

**Discussion by walle-r** [Leon406/ToolsFx#269] [General]
![image](https://github.com/Leon406/ToolsFx/assets/48503060/cd5254f7-e794-4183-b599-160b014ea664)
x25519中的点都是压缩的，这里必须输入x,y坐标吗？

**Comment by Leon406** [comment]
目前是这样的

各种ECC曲线算法由boucycastle实现的，未提供相关decompress

**Reply by walle-r** [reply]
好的
